src/main/webapp/app/exercises/shared/plagiarism/plagiarism-sidebar/plagiarism-sidebar.component.ts

```diff
@@ -70,7 +70,7 @@ export class PlagiarismSidebarComponent implements OnChanges {
     }
 
     computeNumberOfPages(totalComparisons: number) {
```

Comment:
Maybe you can rename it to something like maximumPageIndex (and currentPageIndex in other places in the code) or so?